org.bridgedb/src/test/java/org/bridgedb/DataSourcePatternsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *BridgeDb,
+ *An abstraction layer for identifier mapping services, both local and online.
+ *Copyright (c) 2012 Egon Willighagen <egonw@users.sf.net>
+ *Copyright (c) 2012 OpenPhacts
+ *
+ *Licensed under the Apache License, Version 2.0 (the "License");
+ *you may not use this file except in compliance with the License.
+ *You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+ *WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package org.bridgedb;
+
+import java.util.regex.Pattern;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Assertions;
+
+/**
+ * Tests the {@link org.bridgedb.DataSourcePatterns} class
```

Comment:
The first sentence of JavaDoc should always end with a period. Please add these.